nixos/modules/flyingcircus/roles/postgresql.nix

```diff
@@ -219,6 +219,14 @@ in
       };
     };
 
+    services.telegraf.extraConfig = {
+      inputs = {
+        postgresql = {
+          address = "host=/tmp user=root sslmode=disable dbname=postgres";
```

Comment:
Could you explain what /tmp does? Does that mean it connects to the socket? Shouldn't the socket be in /run?